e2e-tests/pom.xml

```diff
@@ -25,6 +25,7 @@
         <module>spring-rest-h2-v1</module>
         <module>spring-rest-postgres-column-types</module>
         <module>spring-rest-mysql-column-types</module>
+        <module>spring-rest-mysql-entity</module>
```

Comment:
why creating a new module just for this one test?

Reply:
I merged both projects